frontend/src/lib/stream-throttle.js

```diff
@@ -0,0 +1,105 @@
+const ACTIVE_SESSION_LENGTH = 30;
+const FROZEN_SESSION_LENGTH = 30;
```

Comment:
Defining it in single place give us possibility to easily change duration of active or frozen part of basic streaming. Also we may want make active session longer or shorter than frozen session (this is because they are defined such way). Also, as mobile app have exact same file, it will be easily changed for both apps.